src/app/components/cases/CreateForm/scaffold.tsx

```diff
@@ -25,6 +26,118 @@ export default (bagId: Components.Schemas.Address["bag_id"], themes?: Components
         isRequired: true
       }
     },
+
+    reporter_anonymous: {
+      type: "ShowHide",
+      props: {
+        shouldShow: ({ values: { reason } }: { values: { reason: any } }) => reason.name === "Melding",
+        field: {
+          type: "RadioFields",
+          props: {
+            isRequired: true,
+            name: "reporter_anonymous",
+            label: "Is de melder anoniem?",
+            options: {
+              yes: "Ja, de melder is anoniem",
+              no: "Nee, de melder is niet anoniem"
+            }
+          }
+        }
+      }
+    },
+    reporter_name: {
+      type: "ShowHide",
+      props: {
+        shouldShow: ({ values: { reporter_anonymous } }: { values: { reporter_anonymous: string } }) => reporter_anonymous === "no",
+        field: {
+          type: "TextField",
+          props: {
+            label: "Naam melder",
+            name: "reporter_name",
+            isRequired: true
+          }
+        }
+      }
+    },
+    reporter_phone: {
+      type: "ShowHide",
+      props: {
+        shouldShow: ({ values: { reporter_anonymous } }: { values: { reporter_anonymous: string } }) => reporter_anonymous === "no",
+        field: {
+          type: "NumberField",
```

Comment:
Should we add a `"TelField"` to amsterdam-react-final-forms? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel